exercises/practice/flatten-array/.meta/example.jq

```diff
@@ -0,0 +1 @@
+flatten | map(select(. != null))
```

Comment:
I just discovered this solution is morally wrong. It should be

```suggestion
.array | flatten | map(select(. != null))
```

It works anyway because `flatten` works on objects as well as arrays.

Should this example solution be amended, or rather the test suite?

Reply:
The example is there to show a working solution, not to provide an exemplar/gold star solution. So long as it passes all the tests, it doesn't matter much. If people want a really good solution, they should use mentoring, not the example.